staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go

```diff
@@ -347,7 +347,7 @@ func IsValidPortName(port string) []string {
 // IsValidIP tests that the argument is a valid IP address.
 func IsValidIP(value string) []string {
 	if net.ParseIP(value) == nil {
-		return []string{"must be a valid IP address, (e.g. 10.9.8.7)"}
+		return []string{"must be a valid IP address, (e.g. 10.9.8.7 or 2001::1)"}
```

Comment:
```suggestion
		return []string{"must be a valid IP address, (e.g. 10.9.8.7 or 2001:db8::ffff)"}
```
(using the documentation range from [RFC 3849](https://tools.ietf.org/html/rfc3849))